tools/operator/testclient/operations/installchaincode.go

```diff
@@ -51,39 +54,47 @@ func (i InstallCCUIObject) InstallCC(config inputStructs.Config, tls string) err
 func (i InstallCCUIObject) createInstallCCObjects(ccObject inputStructs.InstallCC, organizations []inputStructs.Organization, tls string) []InstallCCUIObject {
 
 	var installCCObjects []InstallCCUIObject
-	var channelOpt ChannelOptions
-	var deployOpt InstallCCDeployOpt
-	i = InstallCCUIObject{SDK: ccObject.SDK, TransType: "install", TLS: tls, ChainCodeVer: ccObject.ChainCodeVersion, ChainCodeID: ccObject.ChainCodeName}
 	orgNames := strings.Split(ccObject.Organizations, ",")
-	channelOpt = ChannelOptions{OrgName: orgNames}
-	deployOpt = InstallCCDeployOpt{ChainCodePath: ccObject.ChainCodePath, Language: ccObject.Language}
-	deployOpt.MetadataPath = ccObject.MetadataPath
-	i.DeployOpt = deployOpt
-	i.ChannelOpt = channelOpt
-	i.ConnProfilePath = paths.GetConnProfilePath(orgNames, organizations)
+	targetPeers := strings.Split(ccObject.TargetPeers, ",")
```

Comment:
We should revisit how we input and parse arrays. Instead of places in the yaml were we create comma separated lists, they should be actual YAML arrays, and then they could be parsed directly to arrays:

```
targetPeers:
- peer0-org1
- peer0-org2
- peer1-org1
- peer1-org2
```

If this was the structure of our yaml we could parse it directly to a `[]string`, there are other places in the input and network files we do the same thing

Reply:
If you don't want to implement this now, we can open a Jira and do it in another PR on top of this work

Reply:
we are using comma separated list as input in yaml.